src/modules/orders/dto/update-orders.dto.ts

```diff
@@ -1,44 +1,51 @@
-import {ApiProperty} from "@nestjs/swagger";
-import {IsDate, IsEnum, IsNumber, IsOptional, IsString} from "class-validator";
-import {OrderStatus} from "src/common/enums";
+import { ApiProperty } from '@nestjs/swagger';
+
+import {
+  IsDate,
+  IsEnum,
+  IsNumber,
+  IsOptional,
+  IsString,
+} from 'class-validator';
+import { OrderStatus } from 'src/common/enums';
 
 export class UpdateOrderDto {
-    @ApiProperty({ example: '2024-11-01T10:00:00Z' })
-    @IsOptional()
-    @IsDate()
-    readonly collection_date?: Date;
-
-    @ApiProperty({ example: '123 Main St, Springfield' })
-    @IsOptional()
-    @IsString()
-    readonly collection_address?: string;
-
-    @ApiProperty({ example: OrderStatus.COMPLETED, enum: OrderStatus })
-    @IsOptional()
-    @IsEnum(OrderStatus)
-    readonly status?: OrderStatus;
-
-    @ApiProperty({ example: 'Leave at front desk' })
-    @IsOptional()
-    @IsString()
-    readonly note?: string;
-
-    @ApiProperty({ example: 'Delayed delivery' })
-    @IsOptional()
-    @IsString()
-    readonly failed_reason?: string;
-
-    @ApiProperty({ example: 2 })
-    @IsOptional()
-    @IsNumber()
-    readonly routeId?: number;
-
-    @ApiProperty({ example: 2 })
-    @IsOptional()
-    @IsNumber()
-    readonly customerId?: number;
-
-    @ApiProperty({ example: [1, 2, 3] })
-    @IsOptional()
-    readonly luggageIds?: number[];
-}
\ No newline at end of file
+  @ApiProperty({ example: '2024-11-01T10:00:00Z' })
```

Comment:
same, I dont see eslint changes

Reply:
Sorry, I didn't understand the error. Can you explain?

Reply:
Eslint didn't work for me before. Now I connected it and the code looks like this